net-mgmt/pfSense-pkg-arpwatch/Makefile

```diff
@@ -0,0 +1,45 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-arpwatch
+PORTVERSION=	0.1.0
+PORTREVISION=	0
```

Comment:
PORTREVISION line must be removed for new ports or when PORTVERSION changes. It defaults to 0